streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -70,14 +69,15 @@
      * @param id                    the ID of this task
      * @param consumer              the instance of {@link Consumer}
      * @param producer              the instance of {@link Producer}
+     * @param restoreConsumer       the instance of {@link Consumer} used when restoring state
      * @param partitions            the collection of assigned {@link TopicPartition}
      * @param topology              the instance of {@link ProcessorTopology}
      * @param config                the {@link StreamingConfig} specified by the user
      */
-    @SuppressWarnings("unchecked")
```

Comment:
@guozhangwang, I'm not sure what you mean. The base of this PR is 25a869b, which appears to be the latest commit on `streaming`. I can get rid of this PR's removal of line 77 if need be.